src/app/demo/demo.component.html

```diff
@@ -28,5 +28,8 @@ <h4 class="card-title">{{ user.name }}</h4>
     <div class="card-block">
       <button class="btn btn-danger" (click)="signOut()">Sign out</button>
     </div>
+    <div class="card-block">
```

Comment:
Don't really think this needs to be shown in the demo page. If we do, we can show this only when a user is logged in with Google.

Reply:
I have applied `ngIf` to show it only when user is logged by Google provider.